client/scripts/main.js

```diff
@@ -11,7 +11,11 @@ var __listen_to_the_ft = (function(){
 		var stored = localStorage.getItem(storageKey);
 		
 		function save(){
-			localStorage.setItem(storageKey, JSON.stringify( stored ) );
+			try{
+				localStorage.setItem(storageKey, JSON.stringify( stored ) );
```

Comment:
We have a module for this --> https://github.com/matthew-andrews/superstore-sync